config/clusters/nasa-esdis/common.values.yaml

```diff
@@ -38,20 +38,58 @@ jupyterhub:
         authenticator_class: github
       GitHubOAuthenticator:
         allowed_organizations:
-          - nasa-esdis:cloud-users
+          - nasa-veda-workshops:ieee-grss-webinar-mar-2024
         scope:
           - read:org
       Authenticator:
         admin_users:
           - bilts # Patrick Quinn
           - freitagb # Brian Freitag
+          - slesaad # Slesa Adhikari
   singleuser:
-    profileList:
-      - display_name: Python
-        description: Python datascience environment
-        default: true
-        kubespawner_override:
-          image: openscapes/python:6ee57a9
+      defaultUrl: /lab
+      profileList:
+        - display_name: "Modified Pangeo Notebook"
+          slug: modified-pangeo
+          description: Pangeo based notebook with a Python environment
+          default: true
+          kubespawner_override:
+            image: public.ecr.aws/nasa-veda/nasa-veda-singleuser:2024-03-07
+            init_containers:
+              # Need to explicitly fix ownership here, as otherwise these directories will be owned
+              # by root on most NFS filesystems - neither EFS nor Google Filestore support anonuid
+              - name: volume-mount-ownership-fix
+                image: busybox:1.36.1
+                command:
+                  - sh
+                  - -c
+                  - id && chown 1000:1000 /home/jovyan /home/jovyan/shared && ls -lhd /home/jovyan
+                securityContext:
+                  runAsUser: 0
+                volumeMounts:
+                  - name: home
+                    mountPath: /home/jovyan
+                    subPath: "{username}"
+                  # mounted without readonly attribute here,
+                  # so we can chown it appropriately
+                  - name: home
+                    mountPath: /home/jovyan/shared
+                    subPath: _shared
+              # this container uses nbgitpuller to mount https://github.com/NASA-IMPACT/veda-docs/ for user pods
+              # image source: https://github.com/NASA-IMPACT/veda-jh-environments/tree/main/docker-images/base/nasa-veda-singleuser-init
+              - name: nasa-veda-singleuser-init
+                image: public.ecr.aws/nasa-veda/nasa-veda-singleuser-init:38e8998f9be64b0a59ac6c4d6d152d3403121dfc4be6d49bdf52ddc92827af8a
+                command:
+                  - "python3"
+                  - "/opt/k8s-init-container-nb-docs.py"
+                  - "/home/jovyan"
+                volumeMounts:
+                  - name: home
+                    mountPath: /home/jovyan
+                    subPath: "{username}"
+                securityContext:
+                  runAsUser: 1000
+                  runAsGroup: 1000
```

Comment:
```suggestion
    defaultUrl: /lab
    profileList:
      - display_name: "Modified Pangeo Notebook"
        slug: modified-pangeo
        description: Pangeo based notebook with a Python environment
        default: true
        kubespawner_override:
          image: public.ecr.aws/nasa-veda/nasa-veda-singleuser:2024-03-07
          init_containers:
            # Need to explicitly fix ownership here, as otherwise these directories will be owned
            # by root on most NFS filesystems - neither EFS nor Google Filestore support anonuid
            - name: volume-mount-ownership-fix
              image: busybox:1.36.1
              command:
                - sh
                - -c
                - id && chown 1000:1000 /home/jovyan /home/jovyan/shared && ls -lhd /home/jovyan
              securityContext:
                runAsUser: 0
              volumeMounts:
                - name: home
                  mountPath: /home/jovyan
                  subPath: "{username}"
                # mounted without readonly attribute here,
                # so we can chown it appropriately
                - name: home
                  mountPath: /home/jovyan/shared
                  subPath: _shared
            # this container uses nbgitpuller to mount https://github.com/NASA-IMPACT/veda-docs/ for user pods
            # image source: https://github.com/NASA-IMPACT/veda-jh-environments/tree/main/docker-images/base/nasa-veda-singleuser-init
            - name: nasa-veda-singleuser-init
              image: public.ecr.aws/nasa-veda/nasa-veda-singleuser-init:38e8998f9be64b0a59ac6c4d6d152d3403121dfc4be6d49bdf52ddc92827af8a
              command:
                - "python3"
                - "/opt/k8s-init-container-nb-docs.py"
                - "/home/jovyan"
              volumeMounts:
                - name: home
                  mountPath: /home/jovyan
                  subPath: "{username}"
              securityContext:
                runAsUser: 1000
                runAsGroup: 1000
```